libs/inform/src/lib/services/modal/modal.service.ts

```diff
@@ -25,9 +25,14 @@ import { NgxModalAbstractComponent } from '../../abstracts';
 @Injectable({ providedIn: 'root' })
 export class NgxModalService {
 	/**
-	 * A subject to hold the close event
+	 * A subject that keeps track of whether a modal is currently active
 	 */
-	private modalClosedSubject: Subject<void>;
+	private hasModalSubject: BehaviorSubject<boolean> = new BehaviorSubject(false);
```

Comment:
Don't forget the `$` suffix to indicate an rxjs stream.

Reply:
I always use it for any type of rxjs stream. I don't know what the convention is for the codebase, I don't have a strong preference, but let's avoid having Frankenstein code with two or more conventions for the same thing.